hydrolib/core/dflowfm/ini/models.py

```diff
@@ -55,6 +56,15 @@ class Config:
         extra = Extra.ignore
         arbitrary_types_allowed = False
 
+    def __init__(self, **data):
+        super().__init__(**data)
```

Comment:
Wouldn't it be more efficient to first check if there are unknown keywords and then call `super().__init__()`? It seems we have all the data we need to determine if there are unknown keywords. And if there are unknown keywords there is no reason to let pydantic work all its magic, since we are raising an error anyway?

Reply:
I am not sure, but I think it had to do with the exclude fields not yet being initialized, thus always returning an empty list for it, will check this.

Reply:
I checked it by moving the super().__init__() below the raise_error_for_unknown_keywords call. The new tests are still green, so I think we can shift it in that way.

Reply:
Nice! :)

Reply:
discussed, this appears to cause more test to failt, because the exclude_fields of structure does some checks which should be resolved by OO. This causes the check to crash when the exclude_fields is executed before init.

Reply:
I will keep the init before the raise_error_for_unknown_keywords call.